operator/pkg/reconciliation/defaults.go

```diff
@@ -0,0 +1,11 @@
+package reconciliation
+
+var (
+	// Provides reasonable defaults for the logger container.
+	DefaultsLoggerContainer = buildResourceRequirements(500, 256)
```

Comment:
pretty sure these could drop to
```suggestion
	DefaultsLoggerContainer = buildResourceRequirements(100, 64)
```